diff --git a/openerp/addons/document/dict_tools.py b/bin/addons/document/dict_tools.py
index 10d2fdc..49fdcf8 100644
--- a/openerp/addons/document/dict_tools.py
+++ b/openerp/addons/document/dict_tools.py
@@ -4,7 +4,8 @@
 #
 # This program is Free Software; you can redistribute it and/or
 # modify it under the terms of the GNU Lesser General Public License
-# as published by the Free Software Foundation; version 2 of the License.
+# as published by the Free Software Foundation; version 2.1 of the
+# License or any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
@@ -12,8 +13,8 @@
 # GNU Lesser General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+# along with this program; if not, write to the Free Software Foundation
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301 USA
 ###############################################################################
 
 
